### PR TITLE
Spelled out colloquialism "crit"

### PR DESCRIPTION
**Pull request by bleehu**
into critical hit, etc, in classes and weapons docs

 Changes to be committed:
	modified:   CharacterCreation/02_Classes.txt
	modified:   Items/04_00_WeaponAttachments.txt
	modified:   Items/04_01_Melee_&_Thrown_Weapons.txt

### CODE REVIEW

File: Items/04_00_WeaponAttachments.txt [hunk 1]
```diff
@@ -606,9 +606,9 @@ round where it's gonna hurt.
 
     Vorpal rounds deal an extra 10% of normal weapon damage on critical hit and
 20% damage on extra-crits. For example, if your gun deals 80 damage normally,
-and you roll a crit when using Vorpal rounds, your gun deals 80 + 10 (for crit)
-+ 8 (Vorpal damage) = 98. If you scored an extra-crit with the same weapon, 
-you'd do 80 + 20 (extra crit damage) + 16 = 116 damage. 
+and you roll a critical hit when using Vorpal rounds, your gun deals 80 + 10 
+(for crit) + 8 (Vorpal damage) = 98. If you scored an extra-crit with the same
```
Comment (Turtlelord26) [inline Items/04_00_WeaponAttachments.txt:610]: I think we also want extra-crit to be spelled out to extra-critical?

**Review by Turtlelord26 — CHANGES_REQUESTED** [review]
Looks mostly excellent. 👍

**Comment by Turtlelord26** [comment]
Found the changes that were blocking automatic merge, thought I'd take care of that since I made them.

File: Items/04_00_WeaponAttachments.txt (hunk 1 above)
Comment (bleehu) [reply Items/04_00_WeaponAttachments.txt:610]: So we actually don't define "extra-critical" but instead formally define "extra-crit" in the combat rules...

Comment (Turtlelord26) [reply Items/04_00_WeaponAttachments.txt:610]: Oh. That's a consistency problem. We should change that. But it's also different enough that we can classify it as a new issue.

**Review by Turtlelord26 — APPROVED** [review]

**Pull request merged**